tests/platforms/macOS/app/test_package.py

```diff
@@ -550,8 +459,8 @@ def test_dmg_with_installer_icon(package_command, first_app_with_binaries, tmp_p
     # Package the app without signing or notarization
     package_command.package_app(
         first_app_with_binaries,
-        sign_app=False,
         notarize_app=False,
+        adhoc_sign=True,
```

Comment:
To confirm - this test has gone from "no signing" to "adhoc signing" - which is fine, given the discussion we've had; but we need to make sure we've disabled/mocked the actual signing code, or else this is going to massively slow down the tests. Can you confirm these have been mocked?